config/jobs/cert-manager-webhook-example/cert-manager-webhook-example-presubmits.yaml

```diff
@@ -1,5 +1,5 @@
 presubmits:
-  jetstack/cert-manager-webhook-example:
```

Comment:
Can we move this to be a subdirectory of https://github.com/jetstack/testing/tree/master/config/jobs/cert-manager instead? We could then also look to collate cert-manager ones into its own subdirectory there too

Reply:
That seems sane